base/applications/charmap/map.c

```diff
@@ -260,6 +260,7 @@ SetFont(PMAP infoPtr,
         {
             if (out[i] != 0xffff)
             {
+                if (ch[i] == ' ') continue;
```

Comment:
This is acceptable for simplicity. But really, we know none of the characters before space are printable either, so we could just change the loop boundaries to start at `' ' + 1`.

Reply:
Sure, I've started the loop at ' ' + 1